manta-pay/src/lib.rs

```diff
@@ -41,6 +41,10 @@ pub mod key;
 #[cfg_attr(doc_cfg, doc(cfg(all(feature = "groth16", feature = "test"))))]
 pub mod parameters;
 
+#[cfg(all(feature = "groth16", feature = "test"))]
+#[cfg_attr(doc_cfg, doc(cfg(all(feature = "groth16", feature = "test"))))]
+pub mod payment;
```

Comment:
Can we move this module into `test`? I think the name is very misleading at the top level.

Reply:
Also `assert_valid_proof` should at least be moved into `manta_accounting::transfer::test`.

Reply:
> Can we move this module into `test`? I think the name is very misleading at the top level.

After moving into `test`, we cannot import this module in `manta-benchmark`. Should we rename it as `sample_payment.rs`?

Reply:
> Also `assert_valid_proof` should at least be moved into `manta_accounting::transfer::test`.

`assert_valid_proof` requires `config` in `manta-pay`. Moving this function to `manta-accounting` will introduce cycle dependency.

Reply:
No, generalize it to using `transfer::Configuration`.